website/content/en/contribute/_index.md

```diff
@@ -25,6 +25,10 @@ There’s everything from code-specific issues to documentation issues and every
 
 This may seem scary at first, especially if you’re new, but it’s a great way to dive in and see friendly faces after chatting with folks on Slack when you’re more comfortable. The meetings are structured with talking points in a shared google doc. You may be given time to introduce yourself (if you want!) and you are welcome to bring a topic to the meeting.
 
+**Contribute a new idea**
+
+While there are always a number of ongoing topics in the TAG, they all started as new ideas. If you have a new proposal we encourage you to [open an issue using the `community contribution` template](https://github.com/cncf/tag-app-delivery/issues/new) and start to build the community of people who would want to work with you on it!
```

Comment:
Does this still need to be added on GH side? Or will this happen with merging this PR?
We could also link directly to [this wizardy form](https://github.com/abangser/tag-app-delivery/issues/new?assignees=&labels=proposals&projects=&template=community-contribution.yml&title=%5BCommunity%5D+%3CShort+description%3E) (which I now realise lives on your repo)

Reply:
This is only on my repo because I wanted to make it visible before we merged to main here. By merging my commits to main on my fork I am able to preview the changes before we need to merge them here. I am not sure of any other way to do the preview.

This was the lesson I learned from the first merge where everything _looked_ OK as yaml but still had errors once on main and actually applied to the GitHub Issues tab!

Reply:
Gotcha. Right now, if you click on the link, you just get to the New Issue page with no way to choose the "community contribution template".
This can be quite confusing. Is the expectation, that as soon as we merge the PR, it will be possible to choose the template from https://github.com/cncf/tag-app-delivery/issues/new?